sdk/core/azure-mgmt-datalake-nspkg/setup.py

```diff
@@ -34,7 +34,7 @@
     name='azure-mgmt-datalake-nspkg',
     version='3.0.1',
     description='Microsoft Azure Data Lake Management Namespace Package [Internal]',
-    long_description=open('README.rst', 'r').read(),
+    long_description=open('README.md', 'r').read(),
```

Comment:
Need to add the long description type

Reply:
Could you explain what you mean? I see it used in a similar way [here](https://github.com/Azure/azure-sdk-for-python/blob/90fe8d9c0263ba739d6dad0bdd270707edbc141a/sdk/keyvault/azure-keyvault-nspkg/setup.py#L29)

Reply:
you need
long_description_content_type="text/markdown",